exporter/cortexexporter/DESIGN.md

```diff
@@ -0,0 +1,287 @@
+
+# **OpenTelemetry Collector Cortex Exporter Design**
+
+Authors: @huyan0, @danielbang907
+
+Date: July 30, 2020
+
+## **1. Introduction**

```

Comment:
Some "nooby" question here:
* Does "Prometheus" allow us to write using the same API directly to its storage? If yes I am so happy to remove the prometheus exporter that we have.

Reply:
I don't think so. Prometheus is pull-based and it scrapes data from exporters, and the text exposition format that Prometheus use to ingest data is different from the remote write API.